core/storage/user/gae_models.py

```diff
@@ -2228,6 +2226,9 @@ class UserAuthModel(base_models.BaseModel):
     # A code associated with profiles on android to provide a PIN based
     # authentication within the account.
     pin = ndb.StringProperty(default=None)
+    # User id of the full user which the profile user is associated with.
```

Comment:
Perhaps explain in this comment why this info needs to be in this model (at first glance it has nothing to do with auth).

Reply:
Added more info now for the explaination.

Reply:
See earlier comment for how to rewrite this.

Also -- this comment explains what the field is, but still doesn't explain why this is relevant for auth. Could you explain further?